tmva/sofie_parsers/src/RModelParser_ONNX.cxx

```diff
@@ -159,6 +159,43 @@ std::unique_ptr<ROperator> make_ROperator_Transpose(const onnx::NodeProto& nodep
    return op;
 }
 
+
+std::unique_ptr<ROperator> make_ROperator_Max(const onnx::NodeProto& nodeproto, const onnx::GraphProto& /*graphproto */, std::unordered_map<std::string, ETensorType>& tensor_type){
+
+   ETensorType input_type = ETensorType::UNDEFINED;
+
+   for (int i = 0; i < 2; ++i) {
+      auto input_name = nodeproto.input(i);
+      auto it = tensor_type.find(input_name);
+      if (it != tensor_type.end()){
+         // according to ONNX both inputs have same time
+         if (i == 0) input_type = it->second;
+         else
+            assert(it->second == input_type);
+      } else {
+         throw std::runtime_error("TMVA::SOFIE ONNX Parser Max op has input tensor" + input_name + " but its type is not yet registered");
+      }
+   }
+
+   std::unique_ptr<ROperator> op;
+
+   switch(input_type){
+   case ETensorType::FLOAT:
+      op.reset(new ROperator_Max<float>(nodeproto.input(0), nodeproto.input(1), nodeproto.output(0)));
```

Comment:
You should change the signature for constructor ROPerator_Max (see comment above)